src/main/tray.ts

```diff
@@ -16,7 +16,7 @@ import { isDevelopment } from "../common/vars";
 export let tray: Tray;
 
 // refresh icon when MacOS dark/light theme has changed
-nativeTheme.on("updated", () => tray?.setImage(getTrayIcon()));
+nativeTheme?.on("updated", () => tray?.setImage(getTrayIcon()));
```

Comment:
`nativeTheme` not available in some OS-es?

Reply:
Now that windowManager is exposed to both main/renderer this needs to be protected, `nativeTheme` is undefined in renderer :/

Reply:
Then probably better to move inside `initTray(windowManager: WindowManager)`